fe/fe-core/src/main/java/com/starrocks/external/RemoteFileOperations.java

```diff
@@ -78,6 +83,14 @@ public List<RemoteFileInfo> getPresentFilesInCache(Collection<Partition> partiti
         return fillFileInfo(presentFiles, pathKeyToPartition);
     }
 
+    public List<RemoteFileInfo> getRemoteFileInfoForStats(Collection<Partition> partitions, Optional<String> hudiTableLocation) {
```

Comment:
It's really weird that `RemoteFileOperations` provides special API for statistics calculator. I think it only provide `get` or `getIfPresent`.

Reply:
discussed